add_user.php

```diff
@@ -28,47 +28,60 @@
 			<!-- import the webpage's javascript -->
 			<!-- nav javascript -->
 			<script src="js/nav.js" defer></script>
```

Comment:
#used javascript for responsive navigation bar to burger menu, show password, and back to top button